_appmap/test/test_django.py

```diff
@@ -213,7 +213,7 @@ def test_enabled(self, pytester):
     def test_disabled(self, pytester, monkeypatch):
         monkeypatch.setenv("_APPMAP", "false")
         result = pytester.runpytest("-svv", "-p", "no:randomly", "-k", "test_request")
-        result.assert_outcomes(passed=1, failed=0, errors=0)
+        result.assert_outcomes(passed=2, failed=0, errors=0)
```

Comment:
This also needs to check that no request recordings were created.

Reply:
Doesn't the following line assert that as well?

https://github.com/getappmap/appmap-python/blob/f90ca457a33bdb9cd4eb808b3da27e9accb3e2d4/_appmap/test/test_django.py#L217

Reply:
Well, it does, in the case that `APPMAP=false` (i.e. recording is completely disabled). The test I asked for ensures that request recordings aren't generated when other AppMaps are.